sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/util/Utilities.java

```diff
@@ -148,12 +149,21 @@ public static <T> T[] arraySlice(T[] data, int start) {
     }
 
     public static void runProcess(String directory, String... commands) throws IOException, InterruptedException {
-        ProcessBuilder processBuilder = new ProcessBuilder();
-        processBuilder.command(commands);
-        processBuilder.directory(new File(directory));
-        processBuilder.inheritIO();
+        File out = File.createTempFile("out", ".tmp", new File("."));
+        out.deleteOnExit();
+        ProcessBuilder processBuilder = new ProcessBuilder()
+                .command(commands)
+                // If this is called from a JUNIT test the output
+                // of the process interferes with the surefire plugin communication,
+                // so we need to redirect the output.
+                .directory(new File(directory))
+                .redirectOutput(out)
+                .redirectError(out);
         Process process = processBuilder.start();
         int exitCode = process.waitFor();
+        List<String> strings = Files.readAllLines(out.toPath());
+        for (String s: strings)
+            System.out.println(s);
```

Comment:
Won't it be better to print this only if exitCode != 0? The CI logs are long enough already.